### PR TITLE
feat(ImageRadioItem): Add ImageRadioItem component

### PR DESCRIPTION
**Pull request by raulsteurer**
Also includes a ImageRadioGroup utility feature

### CODE REVIEW

**Comment by lukasbals** [comment]
Can we name this component a little bit more generic? I think if a user uses a different image than a map this can also be used as something else than a `MapTypeCard`? 

Maybe something like `ImageRadioItem` and `ImageRadioGroup` would be better? What do you think?

**Comment by lukasbals** [comment]
Another question: Can't we implement a working example for the `ImageRadioGroup`/`MapTypeCardGroup` in Storybook? This would be super nice!

File: src/components/MapTypeCard/MapTypeCard.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    black,
+    primary500,
+    primary800,
+  });
+
+  const isActivated = isSelected || value === currentValue;
+
+  useEffect(() => {
+    controls.start(isDisabled ? 'disabled' : isActivated ? 'selected' : 'default');
+  }, [isDisabled]);
```
Comment (lukasbals) [inline src/components/MapTypeCard/MapTypeCard.tsx:43]: Shall we disable those warnings?

**Comment by tobsut97** [comment]
- Could you please remove the "bounciness" in the animation and just make it a smooth transition? 
- The label for the active type should be `primary/800`.

Apart from these two things: looking good!

File: src/components/MapTypeCardGroup/MapTypeCardGroup.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+1. A `value` prop with the value of the MapTypeCard that is to be selected by default.
+
+## Usage
+
+```typescript
+const MyFeature = () => {
+  const [value, setValue] = useState('value1');
+  return (
+    <MapTypeCardGroup onChange={setValue} value={value}>
+      <MapTypeCard thumbnail="urlToThumbnail" value="value1" title="Option 1" />
```
Comment (lukasbals) [inline src/components/MapTypeCardGroup/MapTypeCardGroup.stories.mdx:20]: I think the thumbnail has to be a `ReactNode`, right?

File: src/components/MapTypeCardGroup/MapTypeCardGroup.test.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+const defaultProps: MapTypeCardGroupProps = {
+  onChange: onChangeSpy,
+  value: 'value1',
+};
+
+const setup = (props: Partial<MapTypeCardGroupProps> = {}) => {
+  const combinedProps = { ...defaultProps, ...props };
+  render(
+    <MapTypeCardGroup {...combinedProps}>
+      <MapTypeCard thumbnail={<></>} value="value1" title="Card 1" />
```
Comment (lukasbals) [inline src/components/MapTypeCardGroup/MapTypeCardGroup.test.tsx:17]: In the other tests, you used real ``. I think we should stay consistent.

**Comment by lukasbals** [comment]
Please rename the commit message since you renamed the component you built.

**Comment by tobsut97** [comment]
@raulsteurer Coolio, bounciness is removed as far as I can see. Still it'd be nice if you could please shorten the duration of the border transition. 100ms should do it.

File: src/components/ImageRadioItem/ImageRadioItem.tsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+      controls.start('selected');
+      return;
+    }
+    controls.start('default');
+  };
+
+  const onClickHandler = () => (onChange ? onChange(value) : onClick ? onClick(value) : null);
+
+  return (
+    <MotionConfig transition={{ bounce: false, ease: false, duration: 0.1 }}>
```
Comment (lukasbals) [inline src/components/ImageRadioItem/ImageRadioItem.tsx:71]: Can't you use one of the constants for the duration? E.g.: `MEDIUM_TRANSITION_DURATION`

**Review by lukasbals — APPROVED** [review]

**Pull request merged**